CHANGELOG.txt

```diff
@@ -27,6 +28,7 @@ Changelog
  * Fix: Make sure the focus outline of checkboxes is fully around the outer border (Steven Steinwand)
 
 
+
```

Comment:
no need to update the changelog, that is usually done when the PR mergeds, otherwise there will be conflicts on this file as other PRs get merged in.